bluemira/base/parameter_frame/_frame.py

```diff
@@ -406,8 +406,18 @@ def _validate_units(param_data: Dict, value_type: Iterable[Type]):
     try:
         quantity = pint.Quantity(param_data["value"], param_data["unit"])
     except ValueError:
-        quantity = pint.Quantity(f'{param_data["value"]}*{param_data["unit"]}')
-        param_data["value"] = quantity.magnitude
+        try:
+            quantity = pint.Quantity(f'{param_data["value"]}*{param_data["unit"]}')
+        except pint.errors.PintError as pe:
+            if param_data["value"] is None:
+                quantity = pint.Quantity(
+                    1 if param_data["unit"] in (None, "") else param_data["unit"]
+                )
+                param_data["source"] = f"{param_data.get('source', '')}\nMAD UNIT 🤯 😭:"
```

Comment:
I worry the emoticons will cause `ParameterFrame` `__repr__` and printing to break...

Reply:
they dont, they will just print as undefined squares if you dont have utf8 support, we already have one somewhere else in the code